terraform/integration/backend/main.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  required_version = "~>1.9.1"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>5.0"
+    }
+  }
+
+  backend "s3" {
+    bucket         = "prsdb-tfstate-integration"
+    dynamodb_table = "tfstate-lock-integration"
+    encrypt        = true
+    key            = "prsdb-infra-integration"
+    region         = "eu-west-2"
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
+}
+
+module "backend" {

```

Comment:
this briefly confused me, as while Terraform backends are a well defined thing, it's also quite common parlance as an antonym for `frontend`, when describing applications. 

To avoid this confusion, I would consider renaming this module `terraform-backend` - what do you think?

Reply:
Sounds sensible to me